ui/src/main/resources/edu/wpi/grip/ui/codegeneration/python/PipelineJevoisRunner.vm

```diff
@@ -0,0 +1,29 @@
+#set($enums = [])
+#set($lines = false)
+import libjevois as jevois
+import cv2
+import numpy as np
+import $moduleName
+
+class GRIPRunner:
+
+    def __init__(self):
+        self.pipeline = $moduleName.$className()
+
+    def process(self, inframe, outframe = None):
+        # Execute the pipeline
+        img = inframe.getCvBGR()
+        self.pipeline.process(img)
+
+        # stream to USB output if current mapping is setup for that
+        if outframe is not None:
+#foreach($step in $pipeline.getSteps())#if($foreach.last)#foreach($output in $step.getOutputs())
+#set($oName = $tMeth.pyName($output.name()))
+#if($oName.contains("contour") || $oName.contains("hull"))
+            cv2.drawContours(img,self.pipeline.$tMeth.pyName($output.name()), -1, (0,255,0),2)
+#end#end#end#end
+            outframe.sendCvBGR(img)
+
+# TODO:  send contour report over USB
+# TODO:  add parameter(s) to enable/disable individual values of the report over USB
```

Comment:
Do you plan to do this before this is merged?

Reply:
I'd like to yes - I also have some refactoring in the template I'd like to include (using socket type rather than name to determine whether or not to generate the drawing code).  I'll amend the commit over the next couple of days; please let me know if any other thins to look at as well.